lib/windows-utilities.js

```diff
@@ -13,9 +13,9 @@ const streamEventLog = (config) => {
         const winEvent = new WinEventLogProcessor({
             events: config.winevent
         });
-        log('Streaming logs from ' + config.winevent.join(', ') + ' events');
+        log('streaming logs from ' + config.winevent.join(', ') + ' events');
         winEvent.on('data', (logObjects) => {
-            debug('Number of log objects found: ' + logObjects.length);
+            debug('# of log objects found: ' + logObjects.length);
```

Comment:
Don't appreciate this shorthand in a log statement - seems lazy and for what reason?

Reply:
Done!